### ISSUE TITLE
maybe tweak dax tools that call sherpa ignore/notice

### ISSUE THREAD

**Issue by DougBurke** [cxcsds/ciao-contrib#644]
With 4.15 beta2 I just tried a 2D fit from dax and got the following output - note the two `dataset 1:` lines. These will come from `ignore2d` / `notice2d` calls. 

```
# -------------------

Mon Oct 24 11:59:21 AM EDT 2022

***
*** No source regions **selected**.  Using data combined from all source regions : 
***   circle(4086.68,4105.6973,2.3862664)
***
  (1/3) Getting data
  (2/3) Getting moments to provide better guess
  (3/3) Doing fit
dataset 1: Field() -> no data
dataset 1: no data -> Circle(4086.68,4105.7,2.38627)
```

I suggest wrapping them in `SherpaVerbosity` - something like (untested)

```
from sherpa.utils.logging import SherpaVerbosity

SherpaVerbosity('WARN'):
    ignore2d()
    notice2d(...)
```

This should probably also be done for the spectral stuff too.

**Comment by kglotfelty** [comment]
just checking ... just wrap the `notice` and `ignore` commands?  Not `fit` because we don't get the fit results printed.

**Comment by DougBurke** [comment]
Yeah - the ignore*/notice* calls will create logging output in 4.15, so we
can turn it off with SherpaVerbosity. There's been no change to the
output/behavior of fit.

**Comment by kglotfelty** [comment]
closed by #673
